### ISSUE TITLE
Does not map both _id and id

### ISSUE THREAD

**Issue by danielstaleiny** [mongoosastic/mongoosastic#283]
Hey, 
I have issue with mapping two ids in one model. I have one _id which is generated automatically with mongo and one which is referring to my own id. I specify `es_indexed: true` on id but only _id is mapped and insert to index 
model definition: 
```
{
        id: { type: String, required: true, es_indexed: true },
	name: { type: String, required: true, es_indexed: true }
}
```
mongo:
```
    "_id" : ObjectId("589312b802366f2414eabdd9"),
    "name" : "Farfetch",
    "id" : "4afc815f-d51f-4ae3-bbd8-8fb54e520eed"
```

Current version: 4.2.4
```
if (prefix === '' && (field === 'id' || field === '_id')) {
      continue
    }
```
Elastic search results:
```
 "_index": "models",
            "_type": "model",
            "_id": "mongoId", 
            "_score": 1,
            "_source": {
               "name": "any name"
            }
```

**After I fix this line of code it mapped to es as I desire.** 
```
if (prefix === '' &&  field === '_id') { continue }
```
Elastic search results:
```
 "_index": "models",
            "_type": "model",
            "_id": "mongoId", 
            "_score": 1,
            "_source": {
               "id": "myOwnId",
               "name": "any name"
            }
```

An alternative solution to this can be some flag force to map this to obey if above.
`es_force_index: true` ? 

this was also open with issue regarding the very same line of code 
[#215](https://github.com/mongoosastic/mongoosastic/issues/215)

**Comment by danielstaleiny** [comment]
@isaymeorg

**Comment by isayme** [comment]
closed by #289
